### DISCUSSION TITLE
Difficulty in making the numbers be pronounced in Portuguese (Brazil).

### DISCUSSION THREAD

**Discussion by bolder004** [SWivid/F5-TTS#646] [Q&A]
Since I'm new to this, I'm not quite sure how to do it. Could someone help me with this? I'm in the final phase of training in Portuguese (Brazil) and have achieved great results, but for the numbers, they are only pronounced in English. This was the first model I ever trained, so I am unsure how to proceed with this.  

https://github.com/user-attachments/assets/d48b8abb-2828-4dd7-b89d-edaefa5052d8

**Comment by murilocurti** [comment]
Can you share your fine tuning process? I wanto to work o this too, but I don't any glue on how to start it

**Reply by bolder004** [reply]
"I can't make this available as it is not very good and I have some issues with it, so I prefer not to share it for now. But soon I will post one in Portuguese/English. I think that way it will be perfect.

**Comment by Tribuzeus** [comment]
Number pronunciations, even in other TTS services, work better when typed in full. 430 = four hundred and thirty. Acronyms too. R$, ºC, US$, etc.

**Reply by bolder004** [reply]
As the one in Spanish has the pronunciation in Spanish, I thought it would be nice if the Portuguese one was like that too. That's why I asked for help, but I'll follow your advice.

**Reply by krc1983** [reply]
Im replying to your post just so I get to know when you are going to deliver your model. thanks._
